src/compute_offsets.cpp

```diff
@@ -43,9 +43,9 @@ void compute_offsets(TypedAST::IfElseStatement* ast, int frame_offset) {
 }
 
 void compute_offsets(TypedAST::CallExpression* ast, int frame_offset) {
-	compute_offsets(ast->m_callee, frame_offset);
+	compute_offsets(ast->m_callee, frame_offset++);
 	for (auto& arg : ast->m_args)
-		compute_offsets(arg, frame_offset);
+		compute_offsets(arg, frame_offset++);
 }
```

Comment:
No need to presume a fix, write a test for it. The one in #232 specifically. You could merge #231, if you haven't already, and it would be super easy to do so. Add it to `tests/function.jp`.

Reply:
Oh, I tested it. I said presumably because there might be other bugs of the same nature roaming.